MetaMask/metamask-extension
# chore: btc: add portfolio ephemeral domain to keyring snap permissions

## **Description**
Adds Metamask portfolio ephemeral domain to list of whiltelisted addresses for keyring snap permissions.

The purpose is to allow testing of the BTC feature on the portfolio's ephemeral builds during development.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26091?quickstart=1)

## **Manual testing steps**

1. Visit portfolio ephemeral domain
2. Confirm when using BTC feature that the balance method can be called on the btc snap

## **Pre-merge reviewer checklist**
